types/testingutils/ssv_msgs.go

```diff
@@ -801,7 +809,7 @@ var TestContributionProofWithJustificationsConsensusData = func(ks *TestKeySet)
 
 	return &types.ConsensusData{
 		Duty:                       TestingSyncCommitteeContributionDuty,
-		Version:                    spec.DataVersionBellatrix,
+		Version:                    spec.DataVersionDeneb,
```

Comment:
IMO the version should be reverted to `DataVersionBellatrix`
The duty not related to proposal here